tests/js/jest.storybook-config.js

```diff
@@ -0,0 +1,6 @@
+const jestConfig = require( './jest.config' );
+
+module.exports = {
+	...jestConfig,
+	testMatch: [ '<rootDir>/.storybook/?(*.)test.js' ],
```

Comment:
This seemed like the easiest solution

We need to include this path in `testMatch` to run the test AFAIK *BUT* we don't want this test ran when running them all normally

I'm sure there are other ways to do this!